test/lib/global_uid_test.rb

```diff
@@ -469,19 +471,39 @@ def with_timed_out_connection(server:, end_time:)
 
     describe "with per-process_affinity" do
       before do
-        GlobalUid::Base.global_uid_options[:per_process_affinity] = true
+        GlobalUid.configuration.per_process_affinity = true
       end
 
-      it "increment sequentially" do
+      it "increment sequentially, selecting a server at random" do
         last_id = 0
         10.times do
           this_id = WithGlobalUID.create!.id
           assert_operator this_id, :>, last_id
         end
+
+        refute_empty(GlobalUid::Base.servers[0].allocator.recent_allocations)
+        assert_empty(GlobalUid::Base.servers[1].allocator.recent_allocations)
```

Comment:
If the server is selected at random, how can we make an assumption about which servers allocated ids?

Reply:
The spec is right, the descriptions are wrong. My [comment below](https://github.com/zendesk/global_uid/pull/70/commits/a36dcf060a1a5e650584c892f946227f584591fb#r412924249) applies here too